### PR TITLE
bettcollins

### PR DESCRIPTION
**Pull request by reecevinto**

### CODE REVIEW

File: bettCollins/collins.md
Comment (Certifieddonnie) [inline bettCollins/collins.md:1]: Okay Nice README 💯 . You explained your code, which I understand. But I don't know how to run it. You might have to include the steps and guides on how a newbie can run the program without asking you.

File: bettCollins/bmicalc.c
Comment (Certifieddonnie) [inline bettCollins/bmicalc.c:1]: Great Code, but no Betty docs styling and C99 styling isn't allowed with Betty.🤷

**Review by Certifieddonnie — CHANGES_REQUESTED** [review]

**Comment by MartyOfMCA** [comment]
@reecevinto Also, I want you to know that your executable is still on this branch. Also, make the name of your directory name lowercase throughout

File: bettCollins/collins.md
Comment (reecevinto) [reply bettCollins/collins.md:1]: okay thank you for the review marty lemme change on that

Comment (reecevinto) [reply bettCollins/collins.md:1]: donnie  i mean , thought it's marty

**Comment by Certifieddonnie** [comment]
I will close this, since you opened a new one👍

**Pull request closed without merging**